README.md

```diff
@@ -4,7 +4,11 @@
 
 This repository provides Dockerfiles for use with Databricks Container Services. These Dockerfiles are meant as a reference and a starting point, enabling users to build their own custom images to suit thier specific needs.
 
-### Documentation
+## Warning: Runtime Incompatibility
+
+The Dockerfiles on the master branch are currently not maintained to be backwards compatible with every Databricks Runtime version, and are not always updated for new versions. The only guarantee of compatibility is for images on [DockerHub](https://hub.docker.com/u/databricksruntime) that are tagged for specific runtime versions.
```

Comment:
I'd leave out the last sentence `The only guarantee of compatibility is for images on [DockerHub](https://hub.docker.com/u/databricksruntime) that are tagged for specific runtime versions.` because it isn't quite true yet. There is Q4 work planned on testing this compatibility - I don't think we can promise it until regular testing occurs